src/main/java/core/basesyntax/service/CSvReader.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class CSvReader extends FileReader {
+    private static final String EXP_HEADER = "type,fruit,quantity";
+    private static final String EXP_FORMAT = "b,";
+
+    @Override
+    public String readFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(fromFileName))) {
+            StringBuilder data = new StringBuilder();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                data.append(line).append(System.lineSeparator());
+            }
+            System.out.println(data.toString().replaceAll(" +", ""));

```

Comment:
replaceAll() - time consuming operation and you are using it 3 times in one method scope. 
I would suggest using local variable here, to calculate once and reuse. 
```suggestion
final var value = data.toString().replaceAll(" +", "");
            System.out.println(data.toString().replaceAll(" +", ""));

```